kmod/igb/igb_main.c

```diff
@@ -1483,13 +1484,16 @@ static void igb_update_mng_vlan(struct igb_adapter *adapter)
 		adapter->mng_vlan_id = IGB_MNG_VLAN_NONE;
 	}
 
-	if ((old_vid != (u16)IGB_MNG_VLAN_NONE) &&
-	    (vid != old_vid) &&
 #ifdef HAVE_VLAN_RX_REGISTER
-	    !vlan_group_get_device(adapter->vlgrp, old_vid)) {
+	old_vid_status = ((old_vid != (u16)IGB_MNG_VLAN_NONE)
+	                  && (vid != old_vid)
+										&& !vlan_group_get_device(adapter->vlgrp, old_vid));
 #else
-	    !test_bit(old_vid, adapter->active_vlans)) {
+  old_vid_status = ((old_vid != (u16)IGB_MNG_VLAN_NONE)
+	                  && (vid != old_vid)
+										&& !test_bit(old_vid, adapter->active_vlans));
```

Comment:
I think we have a tab-vs-space issue going on here. The `&& (vid != old_vid)` line is lined up via spaces, but the following line has a whole lot of tabs that I assume were set at a much smaller value than the kernel standard of 8 spaces per tab. Per kernel standards, indentation should be done solely with tabs and a tab-width of 8. 

The original code was a bit off as well, as it used a combination of tabs and spaces, but if we're changing the lines anyway it's a perfect opportunity to fix it.